tensorflow_serving/model_servers/main.cc

```diff
@@ -191,10 +201,17 @@ int main(int argc, char** argv) {
   tensorflow::string model_name = "default";
   tensorflow::int32 file_system_poll_wait_seconds = 1;
   tensorflow::string model_base_path;
+  tensorflow::string model_version_policy = "LATEST_VERSION";
```

Comment:
Please use the LATEST_VERSION constant, not the string. (Use the enum type here, not the string type.)